src/Randomizer.SMZ3.Tracking/tracker.json

```diff
@@ -471,7 +407,8 @@
       "Column": 6,
       "Row": 4,
       "WhenTracked": {
-        "1": [ "You've got mail!" ]
+        "1": [ "You've got mail!" ],
+        "2": null
```

Comment:
Was this change intentional? Just making sure this wouldn't cause a null error.

Reply:
nvm, looks like it should be fine

Reply:
Yup! I specifically detect `null` and take it to mean "use the default responses"